addons/website_sale/views/templates.xml

```diff
@@ -72,7 +72,7 @@
         <xpath expr="//header//ul[@id='top_menu']/li" position="before">
             <t t-set="website_sale_order" t-value="website.sale_get_order()" />
             <li class="nav-item divider d-none"/> <!-- Make sure the cart and related menus are not folded (see autohideMenu) -->
-            <li id="my_cart" t-attf-class="nav-item#{'' if website_sale_order and website_sale_order.cart_quantity else ' d-none'}">
```

Comment:
Take advantage of the commit to solve another problem. We should use:
```xml
t-attf-class="XXX YYY #{'' if blabla else 'ZZZ'}"
```
instead of 
```xml
t-attf-class="XXX YYY#{'' if blabla else ' ZZZ'}"
```
otherwise we lose the possibility to remove the YYY class with an xpath and it also make the xml less readable for no real good reason.